server/express/passport.js

```diff
@@ -1,12 +1,50 @@
 const passport = require('passport');
 const { LoginStrategy } = require('passport-did-auth')
-const fs = require('fs')
+const dotenv = require('dotenv')
```

Comment:
Is using a `.env` file necessary now to run the express server?

Reply:
No, I first used it to set PEM keys, but it isn't used anymore. I can remove dotenv